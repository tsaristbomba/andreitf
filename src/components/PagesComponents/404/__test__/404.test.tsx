import React from "react"
import renderer from "react-test-renderer"
import Error from "../index"

describe("404", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Error />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
