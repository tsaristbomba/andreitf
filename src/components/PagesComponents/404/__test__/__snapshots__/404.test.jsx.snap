// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `
<div
  className="sc-bdnxRM jETLa-d"
>
  <div
    className="sc-gtsrHT cLkLOX"
  >
    <a
      aria-current="page"
      className="sc-jSFjdj cJRnYe"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      <span>
        T
      </span>
      <span
        className="sc-gKAaRy jlEiya"
      >
        F
      </span>
    </a>
    <h1
      className="sc-dlnjwi dAulQx"
    >
      <div
        className="Typewriter"
        data-testid="typewriter-wrapper"
      />
    </h1>
    <p
      className="sc-hKFxyN hdsYTc"
    >
      The page you're trying to visit 
      <strong>
        does not exist
      </strong>
    </p>
    <a
      aria-current="page"
      className="sc-eCApnc bXplAe"
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
      style={Object {}}
    >
      Get me out!
    </a>
  </div>
</div>
`;
